open-source-software-submissions/editor-in-chief-guide.md

```diff
@@ -1,19 +1,130 @@
+- [ ] **License:** The package has an [OSI approved license](https://opensource.org/licenses)
+- [ ] **Repository:** The repository link resolves correctly
+- [ ] **README:** The package has a README with clear explanation of what the package does and instructions on how to install it along with development instructions. 
+- [ ] **Contributing statement:** The package has a contributing.md file that details how to contribute to the package. 
+- [ ] **Package overlap:** That package doesn't fully overlap with functionality of other packages that have already been submitted to pyOpenSci
+- [ ] **Archive** (JOSS only, may be post-review): The repository DOI resolves correctly
+- [ ] **Version** (JOSS only, may be post-review): Does the release version given match the GitHub release (v1.0.0)?
+
+---
+- [ ] [Initial onboarding survey was filled out ](https://forms.gle/F9mou7S3jhe8DMJ16) 
+We appreciate each maintainer of the package filling out this survey individually. :raised_hands: 
+Thank you authors in advance for setting aside five to ten minutes to do this. It truly helps our organization. :raised_hands:
+---
+
+*******
+
+## Editor comments
+
+
+```
+
+Once the package initial checks are complete, and it is determined that 
+the package is in scope for pyOpenSci review, the EiC will assign an 
+editor to the review. The editor will begin the process of finding reviewers for the package. [Check out the editor guide for more information on this process](editors-guide.md) 
+
+A few final tasks to perform here:
+
+- Add any comments to the bottom of your editor checks comment
+- Update the yaml in the header of the issue with the editor assigned to the review 
+- Add the tag `2/seeking-reviewer(s)` to the issue.
+
+### Submissions that are incomplete or vague: need more details
+
+In some cases online documentation of a package is sparse, the README is 
+minimal or hard to understand or there is no clear documentation setup. Or 
+in other cases some of the YAML at the top of the issue is missing. 
+This makes assessment of the package's scope much harder. 
+In this case, please ask for more information. Even if the package is deemed
+out-of-scope, the package documentation will improve as a result of your questions. 
+
+Example text
+
+```markdown
+Hello <username> and many thanks for your submission.
+
+We are discussing whether the package is in scope and need a bit more 
+information.
+
+Would you mind adding more details and context to your `README` file?
+After reading it, someone with little domain knowledge should understand 
+the aim, goals and functionality of the package.
+
+<optional>
+If a package has overlapping functionality with other packages, we require you 
+to mention in your documentation (README) and in this issue [how it is "best in class"](https://pyopensci.org/policies.html#overlap). Could you add a more detailed comparison to the packages you mention in the README so we can evaluate?
```

Comment:
This link 404s. Should it be an internal link to this section?  
https://www.pyopensci.org/contributing-guide/open-source-software-peer-review/aims-and-scope.html?highlight=overlap#package-overlap

Reply:
oh you know what i just redid a lot of our policies. let me double check on that one to make sure it's correct. i did a lot of work there that might have messed up many links. I'm hoping to just use html proofer for that check in the near future as well it will just be easier.